app/services/legacy/dataset_import_service.rb

```diff
@@ -43,6 +45,10 @@ def dataset_attributes
       location1: build_location,
       licence: build_licence,
       licence_other: build_licence_other,
+      licence_code: build_licence_code,
+      licence_title: licence_info.title,
+      licence_url: licence_info.url,
+      licence_custom: get_extra("licence", default: nil),
```

Comment:
Did you look at the side effect of setting the default to `nil` instead of just here? It seems strange to to have some attributes blank and then this one `nil`.

Reply:
I'm not overly keep on using "" to represent not present, but I guess if it's what we're doing everywhere else.